src/views/ContentReferenceWidget.vue

```diff
@@ -21,36 +21,38 @@
 
 <template>
 	<div v-if="richObject" class="tables-content-widget">
-		<h2>{{ richObject.emoji }}&nbsp;{{ richObject.title }}</h2>
+		<h2>{{ element.emoji }}&nbsp;{{ element.title }}</h2>
```

Comment:
So the `v-if` is still on `richObject` but the content is based on `element`. Is that on purpose?